common/lib/capa/capa/tests/test_capa_problem.py

```diff
@@ -0,0 +1,220 @@
+"""
+Test capa problem.
+"""
+import unittest
+
+from . import new_loncapa_problem
+
+
+class CAPAProblemTest(unittest.TestCase):
+    """ CAPA problem related tests"""
+
+    def test_label_and_description_inside_responsetype(self):
+        """
+        Verify that
+        * label is extracted
+        * description_ids are constructed
+        * <label> tag is removed to avoid duplication
+
+        This is the case when we have a problem with single question or
+        problem with multiple-questions separated as per the new format.
```

Comment:
Is it worth also having a test case with multiple question text (separated per the new format)?